packages/backend-common/config.d.ts

```diff
@@ -20,6 +20,15 @@ export interface Config {
   };
 
   backend: {
+    /** Backend configuration for when request authentication is enabled */
+    authorization?: {
+      /**
+       * Secret shared by all backends for generating tokens
+       * Format is base64 24-bit key
+       */
+      secret?: string;
```

Comment:
Should be marked [secret](https://backstage.io/docs/conf/defining#visibility), right? 😄

Reply:
ah, yes!